thesis-de/content-de.tex

```diff
@@ -769,6 +769,19 @@ \subsection{Frontends}
 
 \subsubsection{Definition eines Graphen}
 \label{definition_eines_graphen}
+Auf der Weboberfläche werden dem User oder dem Administrator einzelne Graphen,
```

Comment:
das ist mehr eine beschreibung was ein graph in unserem projekt ist.

hast du noch das C buch von mir rumfliegen? da gabs eine gute erklärung drinn was ein (mathematischer) graph ist. Eine verbindung von koordinaten auf einem raster oder sowas. eine koordinate ist bei uns eine messung/messwert aus einer timeseries. auf der einen achse dann der tatsächliche wert, auf der anderen der timestamp der aufnahme.